open-metadata-implementation/access-services/glossary-view/glossary-view-server/src/main/java/org/odpi/openmetadata/accessservices/glossaryview/server/service/TermService.java

```diff
@@ -50,6 +50,9 @@
     private static final String USED_IN_CONTEXT_RELATIONSHIP_NAME = "UsedInContext";
     private static final String USED_IN_CONTEXT_RELATIONSHIP_GUID = "2dc524d2-e29f-4186-9081-72ea956c75de";
 
+    private static final String SEMANTIC_ASSIGNMENT_RELATIONSHIP_NAME = "SemanticAssignment";
+    private static final String SEMANTIC_ASSIGNMENT_RELATIONSHIP_GUID = "e6670973-645f-441a-bec7-6f5570345b92";
```

Comment:
I think you can use the helper to load the types by name in order not to hardcode both type name and type guid.